apps/dcos_net/src/dcos_net_mesos_listener.erl

```diff
@@ -21,6 +21,7 @@
 -type task() :: #{
     name => binary(),
     framework => binary() | {id, binary()},
+    container => binary() | undefined,
```

Comment:
Let's rename it to `runtime`, make it required, and use `docker` and `mesos` atoms as values?

```suggestion
    runtime => docker | mesos,
```

Reply:
This doesn't work as certain events such as TASK_UPDATE doesn't have this information.

Reply:
It's expected, `TASK_UPDATE` returns task_status, not task info. In that case we just shouldn't update `runtime` field.